ionoscloud/resource_server.go

```diff
@@ -360,6 +360,11 @@ func resourceServer() *schema.Resource {
 					},
 				},
 			},
+			"label": {
```

Comment:
This is a set, should be plural I think. Also add a description.

Reply:
@cristiGuranIonos for elements of `TypeSet` or `TypeList` I noticed that we use the singular form for  the resource and the plural form for the data source, which makes sense. Also, when referring the resource in the terraform plan I think it's better to do:
```
label {
    key = "labelkey"
    value = "labelvalue"
  }
  label {
    key = "labelkey2"
    value = "labelvalue2"
  }
```
instead of
```
labels {
    key = "labelkey"
    value = "labelvalue"
  }
  labels {
    key = "labelkey2"
    value = "labelvalue2"
  }
```
I think it's fine to leave this as it is. What do you think?

Reply:
hmm actually yes in this case it makes more sense to be `label`.